### DISCUSSION TITLE
channels?

### DISCUSSION THREAD

**Discussion by SoniEx2** [mastodon/mastodon#17967] [General]
it is concerning that some admins believe they are entitled to dictate their users' relationships, and don't understand the harm they're causing. we mean we've only met one such admin but uh yeah that was scary. anyway we believe making the fediverse a network of subnetworks would be cool, and having something like IRC channels where you meet ppl would be cool.

the basic idea is that you'd have to join channels in addition to joining an instance. ofc, you'd still be able to interact with ppl outside channels, but channels by default would provide lots of filtering options.

we feel like a good place to start with something like channels, would be to make boosting sensitive to them. so you only get boosts if you share a channel with the person being boosted. not sure how feasible that would be, tho. thoughts?

**Comment by shleeable** [comment]
This concept is similar to "groups" which are currently being discussed.

**Reply by SoniEx2** [reply]
We feel like this concept is more akin to virtual mobile network operators. Perhaps a better name for it would be "virtual instances"? There's also a whole discussion that can be had about roaming of virtual instances.
